gamemodes/terrortown/gamemode/client/cl_scoring.lua

```diff
@@ -413,16 +392,16 @@ function CLSCORE:BuildHilitePanel(dpanel, title, starttime, endtime)
 	local w, h = dpanel:GetSize()
 
 	-- List of traitors
-	local tr = {}
+	--local tr = {}
 
-	for k, v in pairs(self.Tms) do
-		if k == TEAM_TRAITOR then
-			table.Add(tr, v)
-		end
-	end
+	--for k, v in pairs(self.Tms) do
+	--	if k == TEAM_TRAITOR then
+	--		table.Add(tr, v)
+	--	end
+	--end
```

Comment:
> Please don't look at cl_scoring.lua and cl_scoring_events as I only made them compatible with the new event system without making sure the code is nice. This will be replaced in the roundendscreen PR as well.